### ISSUE TITLE
Docker Compose

### ISSUE THREAD

**Issue by mouzzampk2014** [XanderStrike/goplaxt#33]
Hi, could you please create a docker compose example so I can use with my existing setup. Thank you so much

**Comment by nemchik** [comment]
```yaml
  plaxt:
    container_name: plaxt
    environment:
    - TRAKT_ID=
    - TRAKT_SECRET=
    - ALLOWED_HOSTNAMES=
    - TZ=${TZ}
    image: xanderstrike/goplaxt
    logging:
      driver: json-file
      options:
        max-file: ${DOCKERLOGGING_MAXFILE}
        max-size: ${DOCKERLOGGING_MAXSIZE}
    ports:
    - 8000:8000
    restart: unless-stopped
    volumes:
    - /etc/localtime:/etc/localtime:ro
    - ${DOCKERCONFDIR}/plaxt:/app/keystore
    - ${DOCKERSHAREDDIR}:/shared
```

`ALLOWED_HOSTNAMES` is intentionally blank.

I may keep this open and add it to DS.

**Comment by mouzzampk2014** [comment]
Nice one definitely I was just planning to ping you. Thank you

**Comment by mouzzampk2014** [comment]
> Nice one definitely I was just planning to ping you. Thank you

What are you putting in     - ALLOWED_HOSTNAMES=${DOCKERHOSTNAME}?

**Comment by mouzzampk2014** [comment]
![2](https://user-images.githubusercontent.com/44548525/63625717-592a0c80-c5f8-11e9-9de3-2363c316924b.jpg)

This is what I have or shall I use public ip? But my public ip is dynamic?

**Comment by XanderStrike** [comment]
Feel free to PR that example in.

**Comment by XanderStrike** [comment]
Resolved by #36
